packages/docs/src/components/header/Header.vue

```diff
@@ -116,6 +116,16 @@ export default class Header extends mixins(PropsMixin) {
       flex: 1 1 !important;
       justify-content: space-between;
 
+      & .mr-2{
```

Comment:
mr means margin-right so I think this css is wrong.

Reply:
Ok i have removed it. Thank you for telling me about the error.